cmd/adduser.go

```diff
@@ -311,3 +266,108 @@ func (p *AddUserParams) editUserClaim(c interface{}, ctx ActionCtx) error {
 
 	return nil
 }
+
+type ResponsePermsParams struct {
+	respTTL string
+	respMax string
+	rmResp  bool
+}
+
+func (p *ResponsePermsParams) maxResponseValidator(s string) error {
+	_, err := p.parseMaxResponse(s)
+	return err
+}
+
+func (p *ResponsePermsParams) parseMaxResponse(s string) (int, error) {
+	if s == "" {
+		return 0, nil
+	}
+	v, err := strconv.Atoi(s)
```

Comment:
Feels that this all could be `return strconv.Atoi(s)`(assuming that caller does not do anything with `v` if `err !=nil`.

Reply:
doesn't - if validation fails, it's done